### PR TITLE
Cleanup cmdstream

### PR DESCRIPTION
**Pull request by kusma**
These are just two small-ish cleanups around the command-stream stuff.

### CODE REVIEW

File: src/libhost1x/host1x-gr3d.c [hunk 1]
```diff
@@ -237,12 +237,6 @@ static int host1x_gr3d_reset(struct host1x_gr3d *gr3d)
 	host1x_pushbuf_push(pb, 0x00000000);
 	host1x_pushbuf_push(pb, 0x00000205);
 
-	host1x_pushbuf_push(pb, HOST1X_OPCODE_MASK(0x352, 0x001b));
-	host1x_pushbuf_push(pb, 0x00000000);
-	host1x_pushbuf_push(pb, 0x00000000);
-	host1x_pushbuf_push(pb, 0x41800000);
-	host1x_pushbuf_push(pb, 0x41800000);
```
Comment (digetx) [inline src/libhost1x/host1x-gr3d.c:244]: Maybe initialize viewport to 0x0?

Comment (kusma) [reply src/libhost1x/host1x-gr3d.c:244]: I don't think it makes much sense. The viewport will have to be set to something useful before rendering anyway, leaving it at uninitialized seems easier.

Comment (digetx) [reply src/libhost1x/host1x-gr3d.c:244]: Okay, 0x0 scissor would prevent rendering anyway.

**Review by digetx — APPROVED** [review]
Good to go! 👍

**Pull request merged**